src/components/AppCard/AppsGallery.js

```diff
@@ -122,13 +122,21 @@ const AppsGallery = ({
           }}
         />
       </div>
-      <Grid container spacing={2} style={{ padding: 24 }}>
-        {apps.map(currentApp => (
-          <Grid key={currentApp.id} item xs={12} sm={6} lg={4} xl={3}>
-            <App app={currentApp} key={currentApp.id} />
-          </Grid>
-        ))}
-      </Grid>
+      {/* if apps.length is equal to 0, search error text appears alerting the user there are no results for the query */}
+      {apps.length === 0 ? (
+        <div className='invalid-search'>
+       <i className="fas fa-search"></i><h2 className='invalid-text'>There are no apps matching this search.</h2> 
```

Comment:
I recommend changing the `h2` to a `p` tag instead. `h2` is a bit big

Reply:
Fixed!